res/controllers/Numark-Mixtrack-3-scripts.js

```diff
@@ -1343,13 +1328,55 @@ NumarkMixtrack3.PlayButton = function(channel, control, value, status, group) {
  * *********************************************************************/
 NumarkMixtrack3.BrowseButton = function(channel, control, value, status, group) {
     var shifted  = NumarkMixtrack3.decks.D1.shiftKey || NumarkMixtrack3.decks.D1.shiftKey;
+    var maxview = !NumarkMixtrack3.libraryMode;
+    var LibraryCommand;
+    var LibraryGroup;
+    var expand;
+    var contract;
 
+    switch (skin){
+    case 1: // Deere or Shade
+    LibraryGroup = "[Master]";
+    LibraryCommand = "maximize_library";
+    expand = 1;
+    contract = 0;
+    break;
+
+    case 2: // LateNight
+    LibraryGroup = "[Master]";
+    LibraryCommand = "maximize_library";
+    expand = 1;
+    contract = 0;
+    break;
+
+    case 3: //Dark Metal
+    LibraryGroup = "[Hifi]";
+    LibraryCommand = "show";
+    expand = 0;
+    contract = 1;
+    break;  
+
+    default : 
+    LibraryGroup ="[Master]";
+    LibraryCommand = "maximize_library";
+    }
+    
```

Comment:
Considering this code only works differently for the Dark Metal skin, I think it would make more sense to change the `skin` option to a boolean called `DarkMetal`.